apps/coscrad-frontend/src/utils/test-utils/assertions/assert-element-with-every-id-rendered-for-index.ts

```diff
@@ -0,0 +1,13 @@
+import { assertElementWithTestIdOnScreen } from './assert-element-with-test-id-on-screen';
+
+type HasId<T> = T & {
```

Comment:
I see, this isn't exported

Reply:
Good point. We definitely don't want to export types from tests \ test-utils into production code.

I think the place for this is in a utility-types lib. I've been holding off that, but maybe now is the time. There's certainly a lot of this going on in every lib and every project.